src/internal/components/autosuggest-input/index.tsx

```diff
@@ -212,9 +212,8 @@ const AutosuggestInput = React.forwardRef(
       role: 'combobox',
       'aria-autocomplete': 'list',
       'aria-expanded': expanded,
-      'aria-controls': ariaControls,
```

Comment:
Why remove aria-controls? I think it is required for combobox.

Reply:
The ticket mentioned aria-controls and aria-owns is not necessary, but I added it back now